addons/web_editor/static/src/js/backend/convert_inline.js

```diff
@@ -687,6 +677,28 @@ async function toInline($editable, cssRules, $iframe) {
     flattenBackgroundImages(editable);
     formatTables($editable);
     responsiveToStaticForOutlook(editable);
+    // Fix Outlook image rendering bug.
+    for (const attributeName of ['width', 'height']) {
+        const images = editable.querySelectorAll('img');
+        for (const image of images) {
+            if (image.style[attributeName] !== 'auto') {
+                let value = image.getAttribute(attributeName) || (attributeName === 'height' && image.offsetHeight);
+                if (!value) {
+                    value = attributeName === 'width' ? _getWidth(image) : _getHeight(image);;
+                }
+                if (value) {
```

Comment:
Why not `if` then `else` ?

Reply:
Because it's not the same thing. This is just: `value = a || b; if value then ...` but split for readability (clearly a fail though apparently :-P

Maybe I should have just written the different parts of `||` on separate lines:
```js
const value = image.getAttribute(attributeName) ||
    (attributeName === 'height' && image.offsetHeight) ||
    (attributeName === 'width' ? _getWidth(image) : _getHeight(image));
```
Now again though this code just moved, it's not new :-)